src/NHibernate/AdoNet/ConnectionManager.cs

```diff
@@ -44,7 +43,7 @@ public interface Callback
 		private readonly IInterceptor interceptor;
 
 		[NonSerialized]
-		private bool isFlushing;
+		private bool _releasesEnabled = true;
```

Comment:
On Hibernate side, two additional public methods are there: `enableReleases` and `disableReleases`. They seem unused and I have not port them. They weaken the class in my opinion, since calling them set directly `releasesEnabled` without taking care of on-going flushes.